sdk/mixedreality/Microsoft.Azure.Management.MixedReality/src/Generated/Models/CheckNameAvailabilityResponse.cs

```diff
@@ -32,12 +31,11 @@ public CheckNameAvailabilityResponse()
         /// Initializes a new instance of the CheckNameAvailabilityResponse
         /// class.
         /// </summary>
-        /// <param name="nameAvailable">if name Available. Possible values
-        /// include: 'true', 'false'</param>
+        /// <param name="nameAvailable">if name Available</param>
         /// <param name="reason">Resource Name To Verify. Possible values
         /// include: 'Invalid', 'AlreadyExists'</param>
         /// <param name="message">detail message</param>
-        public CheckNameAvailabilityResponse(string nameAvailable, string reason = default(string), string message = default(string))
+        public CheckNameAvailabilityResponse(bool nameAvailable, string reason = default(string), string message = default(string))
```

Comment:
This introduces a breaking change since we are changing the type of a parameter on a public method.

Please work with the breaking change board to get the Approved-BreakingChange lable to the spec PRs

Reply:
Thanks @m-nash  looked into this a little bit; we are changing this response type to boolean because this is what our service has been returning all along, and this would fix a correctness issue that we've had since the beginning. This should have minimal customer impact because in the last 30 days we have 1622 calls to this API and all but one of them are from the portal (which we already handle the response as boolean and not string), the other call is from an internal microsoft test app.

I have filled out the breaking change template here https://msazure.visualstudio.com/One/_workitems/edit/9314887

Reply:
Makes sense, the approved breaking change label is there now so resolving this